test/recipes/70-test_tls12groupselection.t

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env perl
+
+use OpenSSL::Test::Simple;
+use OpenSSL::Test qw/:DEFAULT srctop_file/;
+use OpenSSL::Test::Utils qw(disabled);
+
+setup("test_tls12groupselection");
+
+plan skip_all => "needs TLSv1.2 enabled"
+    if disabled("tls1_2");
+
+plan skip_all => "needs EC enabled"
```

Comment:
If `ecx` is disabled then it automatically disables `ec`. So it is sufficient to test whether `ecx` is enabled (because if it is, then `ec` must also be)